ts/blueprints/ember-cli-typescript/index.js

```diff
@@ -191,8 +191,9 @@ module.exports = {
     let pkg = JSON.parse(fs.readFileSync(pkgPath, 'utf-8'));
 
     // Really `prepack` and `postpack` would be ideal, but yarn doesn't execute those when publishing
-    this._addScript(pkg.scripts, 'prepublishOnly', 'ember ts:precompile');
-    this._addScript(pkg.scripts, 'postpublish', 'ember ts:clean');
+    // As of https://github.com/yarnpkg/yarn/pull/5712 yarn runs `prepack` and `postpack` when publishing
```

Comment:
Do we still need the comment on 193 any more? 🙂

Reply:
Oh, lol, I meant to delete it!